ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -20,6 +25,24 @@ nginx_sites:
    add_header: "X-Frame-Options SAMEORIGIN"
    access_log: "{{ log_home }}/{{ deploy_env }}-timing.log timing"
    locations:
+    - name: "^~ /(home|impact|pricing|services|software|solutions|askdemo)$"
+      balancer: "{{ deploy_env }}_commcare"
+      proxy_redirect: "http://{{ SITE_HOST }} https://{{ SITE_HOST }}"
+      proxy_next_upstream_tries: 1
+      proxy_read_timeout: 900s
+      proxy_cache: prelogin_cache
+      proxy_cache_use_stale: "error timeout updating http_500 http_502 http_503 http_504"
+      proxy_cache_methods: GET
+      add_header: X-Cache-Status $upstream_cache_status
+    - name: "/lang/"
```

Comment:
Scanning these settings with my eyeballs it looks like they're they same as those above (except for `name`). If I'm not missing an important difference, can this just be added to the regex above?